reporting/impl/src/main/resources/reports/templates/application_list.ftl

```diff
@@ -38,12 +38,14 @@
                 <span class="points">${pointsFromAllTraversal}</span>
                 <span class="legend">story points</span>
             </div>
+            <#if (pointsFromSharedTraversal > 0) >
             <div class="effortPoints shared">
```

Comment:
I think this will make the elements right of this shift to the left.
I suggest to rather add display: hidden, in a way that you add a class pointsCount${pointsFromSharedTraversal?c} and
... .pointsCount0 { display: hidden; }